Formula/l/lla.rb

```diff
@@ -0,0 +1,55 @@
+class Lla < Formula
+  desc "High-performance, extensible alternative to ls"
+  homepage "https://github.com/triyanox/lla"
+  url "https://github.com/triyanox/lla/archive/refs/tags/v0.2.6.tar.gz"
+  sha256 "6fc63f98d4a1da4e6ec231ba17b91b5ca8e472d3aa33fe7ea368c58cd920bf90"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "lla")
+
+    (buildpath/"plugins").each_child do |plugin|
+      next unless plugin.directory?
+
+      plugin_path = plugin/"Cargo.toml"
+      next unless plugin_path.exist?
+
+      system "cargo", "build", "--jobs", ENV.make_jobs.to_s,
+                               "--locked", "--lib", "--release",
+                               "--manifest-path=#{plugin_path}"
+      lib.install plugin/"target/release"/shared_library("lib#{plugin.basename}")
+    end
+  end
+
+  def caveats
+    <<~EOS
+      The Lla plugins have been installed in the following directory:
+        #{opt_lib}
+    EOS
+  end
+
+  test do
+    test_config = testpath/".config/lla/config.toml"
+
+    system bin/"lla", "init"
+
+    output = shell_output("#{bin}/lla config")
+    assert_match "Current configuration at \"#{test_config}\"", output
+
+    system bin/"lla"
+
+    # test lla plugins
+    config_content = test_config.read
+    updated_content = config_content.gsub(/^plugins_dir = ".*"$/, "plugins_dir = \"#{opt_lib}\"")
+    test_config.open("w") do |file|
+      file.write updated_content
+    end
```

Comment:
Actually, we can just do this:
```suggestion
    inreplace(test_config, /^plugins_dir = ".*"$/, "plugins_dir = \"#{opt_lib}\"")
```